Week1/homework/js-exercises/10-compare-arrays.js

```diff
@@ -0,0 +1,19 @@
+// Exercise 10: Compare arrays
+
+'use strict';
+{
+
+let array1 = ['a','b','c','d'];
+let array2 = [1,2,3,4,5,6,7];
```

Comment:
`const` instead of `let` here please since we aren't reassigning variables later in this program